src/query/service/src/spillers/serialize.rs

```diff
@@ -158,7 +160,7 @@ fn bare_blocks_to_parquet<W: Write + Send>(
     let table_schema = infer_table_schema(&data_schema)?;
 
     let props = WriterProperties::builder()
-        .set_compression(Compression::LZ4_RAW)
+        .set_compression(Compression::SNAPPY)
```

Comment:
How much of a performance difference is there between snappy and lz4?

Reply:
It's change by mistake, thanks for the tips.